tools/cockpit.spec

```diff
@@ -93,6 +93,7 @@ Recommends: %{name}-docker = %{version}-%{release}
 Suggests: %{name}-pcp = %{version}-%{release}
 Suggests: %{name}-kubernetes = %{version}-%{release}
 Suggests: %{name}-selinux = %{version}-%{release}
+Suggests: %{name}-pkgupdates = %{version}-%{release}
```

Comment:
Sure. Names a hard ... maybe writing a Feature page will help you figure that out. 

https://github.com/cockpit-project/cockpit/tree/master/doc/guide

The feature pages are meant to help admins imagine how Cockpit is interacting with their system, and to give them clues on how to perform similar actions from the command line.

Once you think such a page through, maybe that'll help with figuring out a name. The package name should have a similar intent. Point toward the thing that we're interacting with, aid in diagnosis, quickly understanding how we're talking to the system.